src/kakarot/interpreter.cairo

```diff
@@ -667,6 +667,9 @@ namespace Interpreter {
         Memory.finalize();
         Stack.finalize();
         State.finalize();
+        with evm {
```

Comment:
could have added the `evm` in the implicit, not a big deal but I feel like it'd make the whole thing lighter.

We can create another issue to put evm in impliciti everywhere if we feel so